tests/noise.py

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright 2023 Matthieu Bouron <matthieu.bouron@gmail.com>
+# Copyright 2023 Nope Foundry
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import array
+
+from pynopegl_utils.misc import SceneCfg, scene
+from pynopegl_utils.tests.cmp_fingerprint import test_fingerprint
+from pynopegl_utils.toolbox.colors import COLORS, get_random_color_buffer
+
+import pynopegl as ngl
+
+
+@test_fingerprint(keyframes=10, tolerance=1)
+@scene()
+def noise_blocky(cfg: SceneCfg):
+    cfg.aspect_ratio = (1, 1)
+    cfg.duration = 10
+
+    return ngl.RenderNoise(
+        type="blocky",
+        frequency=ngl.UniformFloat(value=1.0, live_id="frequency", live_min=0, live_max=10),
+        amplitude=ngl.UniformFloat(value=1.0, live_id="amplitude", live_min=0, live_max=10),
+        octaves=ngl.UniformUInt(value=1, live_id="octaves", live_min=1, live_max=16),
```

Comment:
Btw, is there a limit in the upper value of for loops with certain backends?

Reply:
Added the following limit: [1;8]